cmake/std/atdm/utils/get_known_system_info.sh

```diff
@@ -72,8 +72,7 @@ ATDM_KNOWN_SYSTEM_NAMES_LIST=(
   cts1
   tlcc2
   sems-rhel7
-  sems-rhel6
-  cee-rhel6  # Used for CEE RHEL7 machines as well!
+  cee-rhel7  # Used for CEE RHEL7 machines as well!
```

Comment:
Probably want to remove the comment.

Reply:
Thanks for catching that @jmgate.  I will fix in a new PR.